examples/tldraw-example/package.json

```diff
@@ -24,10 +24,10 @@
     "@types/react-router-dom": "^5.1.8",
     "concurrently": "6.0.1",
     "create-serve": "1.0.1",
-    "dotenv": "^10.0.0",
```

Comment:
Any reason why it is deleted?

Reply:
I had added it because I couldn't figure out how `process.env` was being used in React. But after finding out it is though the build step, I removed it as it was not needed. Will add it back if the project depended on it before. Let me know